libs/langgraph/langgraph/errors.py

```diff
@@ -22,13 +22,22 @@ class GraphRecursionError(RecursionError):
         )
     """
 
-    pass
+    def __init__(
+        self,
+        message: str = "Recursion limit reached.\n\nTroubleshooting URL: https://python.langchain.com/docs/troubleshooting/errors/GRAPH_RECURSION_LIMIT/",
+    ):
+        self.message = message
+        super().__init__(self.message)
 
 
 class InvalidUpdateError(Exception):
     """Raised when attempting to update a channel with an invalid set of updates."""
 
-    pass
+    def __init__(self, message: str, code: Optional[str] = None):
+        if code is not None:
```

Comment:
this is problematic because we don't actually pass the code currently and we do use `message`, so we would lose specific messages we currently have for different invalid updates. for example, here https://github.com/langchain-ai/langgraph/blob/main/libs/langgraph/langgraph/pregel/__init__.py#L848